.github/workflows/test_ert.yml

```diff
@@ -52,4 +52,4 @@ jobs:
 
     - name: Testing...
       run: |
-        make test-ert
+        npm run test-unsafe test/jest/test-ert.test.js
```

Comment:
Nice! I guess we can remove all the old makefile targets? :D

Reply:
Yep! Shall I remove the test shell scripts too?

Reply:
Yes! Thank you! 😄👍